content/docs/dvclive/get-started.md

```diff
@@ -118,23 +119,56 @@ training progress:
 
 <toggle>
 
-<tab title="Standalone report">
+<tab title="Standalone / Notebook">
 
-At the end of each `step`, DVCLive will generate or update a report in
+By default, DVCLive will generate or update a report in
 [`Live.report_file`](/doc/dvclive/api-reference/live#properties) displaying all
-the logged data.
+the logged data:
 
 ![HTML report](/img/dvclive-html.gif)
 
+If you pass `report="notebook"` to DVCLive, the HTML report will be displayed
+and updated inside the output of the cell:
+
+![Notebook report](/img/dvclive-notebook.gif)
```

Comment:
Not sure why, but I'm seeing it with some strange spotted pattern:

https://user-images.githubusercontent.com/2308172/218141432-c860eeec-36c6-48d3-bbfd-672572a5ef66.mov

Reply:
I think it's coming from the original GIF I created with `ffmpeg`

Reply:
Updated the GIF